components/page.js

```diff
@@ -12,9 +31,68 @@ const Content = styled.div`
   padding: 60px;
 `;
 
-export default ({ hackathons, currentPath, children }) => (
-  <Container>
-    <Sidebar currentPath={currentPath} hackathons={hackathons} />
-    <Content>{children}</Content>
-  </Container>
-);
+const HackathonNavBar = styled.div`
+  display: flex;
+  padding-left: 60px;
+`;
+
+const NavItem = styled.a`
+  display: block;
+  cursor: pointer;
+  font-size: 24px;
+  margin-right: 40px;
+  text-decoration: none;
+  color: ${(p) => (p.selected ? COLOR.BLACK : COLOR.DARK_COPY)};
+  border-bottom: ${(p) => (p.selected ? '3px solid black' : 'none')};
```

Comment:
Nit: just for consistency
```suggestion
  border-bottom: ${(p) => (p.selected ? '3px solid COLOR.BLACK' : 'none')};
```